packages/cli/package.json

```diff
@@ -26,7 +26,7 @@
     "dependencies": {
         "@babel/traverse": "^7.22.5",
         "@inquirer/prompts": "^2.3.0",
-        "@nangohq/shared": "^0.36.101",
+        "@nangohq/shared": "^0.37.3",
```

Comment:
unrelated to google-ads but I notice that `shared` version had not been bumped by the publish script. I will investigate and try to understand why